gpu-operator/release-notes.rst

```diff
@@ -115,14 +224,6 @@ Fixed issues
   This fix creates the new service account, a new cluster role, and a new cluster role binding.
   The update prevents failures with Argo CD.
 
-* Previously, duplicate image pull secrets were added to some daemon sets and caused an error
```

Comment:
Note, we added this to the release notes for 23.9.1, but it doesn't look like this fix actually made it into 23.9.1.

Reply:
TY. Hrm, I'll have to figure out why I thought the update made it in.  My MR tracing skills mustn't be what I thought they were.